R/utils.R

```diff
@@ -390,7 +397,7 @@ build_comm_cards <- function(data){
     dplyr::add_row(
       name = 'downloads_trend',
       title = 'Downloads trend',
-      desc = 'Trend of downloads in last 12 months',
+      desc = glue::glue("Trend of downloads in last {amount_months} months"),
```

Comment:
I still think we need to be clear that this is a "monthly" trend, so users know that this value reflects an average increase or decrease in downloads per month. Consider adding the word "Monthly"? Or some other phrase that fits better. I know real estate is tight.

Reply:
Thanks! Updated the title to 'Monthly downloads trend'

![image](https://github.com/pharmaR/riskassessment/assets/8216229/b6e656e6-9d35-4142-a856-9f1d16610f5a)